### PR TITLE
Dev aten garrett

### PR DESCRIPTION
**Pull request by garrettmw**
Minor fixes for Resources to Learn R in the Basics series

### CODE REVIEW

File: tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd [hunk 1]
```diff
@@ -657,7 +720,7 @@ x <- factor(c("yes", "no", "yes"), levels = c("yes", "no"))
 x
```
Comment (donal-at-NEON) [inline tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd:720]: Maybe run the `x` in line 718 (and add spacing as needed), above where you re-arrange them, to clearly show the difference in the output.

File: tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd [hunk 2]
```diff
@@ -136,12 +157,15 @@ library(dplyr)
 library(neonUtilities)
 
 # load the NEON small mammal capture data
+# NOTE: the check.size = TRUE argument means the function 
+# will require confirmation from you that you want to load 
+# the quantity of data requested
 loadData <- loadByProduct(dpID="DP1.10072.001", site = "HARV", 
                  startdate = "2014-01", enddate = "2014-12", 
-                 check.size = T) # Console requires your response!
+                 check.size = TRUE)
```
Comment (donal-at-NEON) [inline tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd:165]: Please add the '# console requires your reponse` comment back to this.

File: tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd [hunk 3]
```diff
@@ -344,9 +368,6 @@ help documentation for this function by running the code: `?group_by()`
 
 ```{r compare-classes}
 
-# View help file for group_by() function
-?group_by()
```
Comment (donal-at-NEON) [inline tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd:348]: Why remove this?

File: tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd [hunk 4]
```diff
@@ -255,7 +270,7 @@ R has 6 atomic vector types.
 By *atomic*, we mean the vector only holds data of a single type.
 
 - **character**: `"a"`, `"swc"`
-- **numeric**: `2`, `15.5`
+- **numeric**: `2`, `15.5`(also called **double**)
```
[the remark this answers is not in the document]
Comment (garrettmw) [reply tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd:273]: This is to set up the reader for when they run typeof() on a numeric vector they made (lines 308-311). The result in the console is "double".

Everything in the [documentation](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/numeric) says that as.double() and as.numeric() are identical. 

_However_, it turns out the reason for the discrepancy is that typeof() returns the _internal_ storage type (which is only really relevant to those with a CS background), and a new student would care more about the R object class (and the class() function) anyway. I removed the parenthetical and changed the function calls to class().

File: tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd [hunk 5]
```diff
@@ -633,21 +688,29 @@ as.character(x)
...
 To convert a factor to a numeric vector, go via a character. Compare
 
-``` {r conv-facor }
+``` {r conv-factor2 }
 
 f <- factor(c(1, 5, 10, 2))
 
-as.numeric(f)  ## wrong!
+as.numeric(f)  ## wrong! returns the assigned integer for each level
+
```
[the remark this answers is not in the document]
Comment (garrettmw) [reply tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd:704]: Good to know, added.

File: tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd (hunk 1 above)
Comment (garrettmw) [reply tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd:720]: Done

File: tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd (hunk 3 above)
Comment (garrettmw) [reply tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd:348]: It opened up a browser tab every time I knit it! Made a separate code chunk with eval=FALSE

File: tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd (hunk 4 above)
[the remark this answers is not in the document]
Comment (donal-at-NEON) [reply tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd]: Good to know! Thanks for the info

File: tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd (hunk 3 above)
Comment (donal-at-NEON) [reply tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd:348]: Ahhh yes that makes sense.

File: tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd [hunk 6]
```diff
@@ -1,16 +1,16 @@
 ---
-syncID: 8346bef7292b46a09a76a0171b05662c
 title: "Getting Started with the R Programming Language"
-description: "This tutorial presents the basics of using R."
-dateCreated: 2014-11-04
-authors: Leah A. Wasser - Adapted from Software Carpentry
+code1: /R/R-basics/R-Basics-Getting-Started.R
 contributors: Donal O'Leary
```
Comment (donal-at-NEON) [inline tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd:4]: Sorry I missed this before. Please add yourself as a contributor to anything that you touched!

File: tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd [hunk 7]
```diff
@@ -374,6 +374,11 @@ class(myData)
 # View class of 'dataBySpSex' object
 class(dataBySpSex)
 
+```
+```{r compare-classes-2, eval=FALSE}
```
Comment (donal-at-NEON) [inline tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd:378]: please add ,comment=NA as a code chunk option to this line

Comment (garrettmw) [reply tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd:378]: Done

File: tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.md [hunk 8]
```diff
@@ -1,113 +1,105 @@
----
```
Comment (donal-at-NEON) [inline tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.md:1]: what happened to the YAML header for this file? this diff shows that it was all removed. I think that you should be able to re-knit this .md file to populate this YAML header again.

File: tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.md [hunk 9]
```diff
@@ -1,198 +1,218 @@
----
```
Comment (donal-at-NEON) [inline tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.md:1]: Looks like we lost this YAML header too. Any idea why/how?

File: tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.md (hunk 8 above)
Comment (garrettmw) [reply tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.md:1]: So I was having an issue where the regular knit button wasn't updating the .R or the .md files at all (only the HTML). I tried running the knit function manually, which didn't work for some reason. I ended up using render() from the rmarkdown package, specifying the file type.

Basically looks like this:
HTML:
  `render(path, output_format = "html_document")`
md:
  `render(path, output_format = "md_document")`
R:
  `knit(path, tangle = TRUE)`            

No idea why the YAML header would be removed. Perhaps there's an argument for that? Will try to investigate a more detailed answer in the morning.

File: tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.md [hunk 10]
```diff
@@ -1,134 +1,145 @@
-R is a versatile, open source programming language that was specifically
-designed for data analysis. R is extremely useful for data management,
-statistics and analyzing data.
+---
```
Comment (donal-at-NEON) [inline tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.md:1]: Good! These YAML headers reappeared :)

**Pull request merged**